src/components/ButtonComponents/NumberButtons/Numbers.js

```diff
@@ -1,16 +1,26 @@
 import React from "react";
 
 //import any components needed
+import NumberButton from "./NumberButton";
 
 //Import your array data to from the provided data file
+import { numbers } from "../../../data";
 
-const Numbers = () => {
+const Numbers = (props) => {
   // STEP 2 - add the imported data to state
+  // const [number, setNumber] = useState();
+
   return (
-    <div>
+    <div className="lowerContainer">
+      {
+        numbers.map(number => NumberButton(number, props))
```

Comment:
While this works, we never want to use our components as functions. We would want to write this as `<NumberButton number={number} {...props}/>`. The `...` is called the spread operator, and using it here is a trick to add all the keys in a object as props.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax